app/lib/common/providers/common_providers.dart

```diff
@@ -73,8 +75,9 @@ final _accountAvatarProvider =
   final account = ref.watch(accountProvider);
   final thumbSize = sdk.api.newThumbSize(48, 48);
   final avatar = await account.avatar(thumbSize);
-  if (avatar.data() != null) {
-    return MemoryImage(avatar.data()!.asTypedList());
```

Comment:
can possibly add comment here too so in case anyone forgets why its intended this way ;)